app/src/main/java/org/opencord/aaa/impl/StateMachine.java

```diff
@@ -357,7 +360,11 @@ protected void setRequestAuthenticator(byte[] authenticator) {
      * @return The state machine identifier.
      */
     public synchronized byte identifier() {
-        identifier = (identifier + 1) % 255;
+        identifier = (identifier + 1);
+        if (identifier == 255) {
```

Comment:
I guess you should be checking identifier == 0 instead. You just want to avoid 0 and 1.

Also, add appropriate comments why we are avoiding them; for what purpose 0 and 1 are being used